.eslintrc.json

```diff
@@ -65,7 +65,13 @@
           "**/*.stories.jsx"
         ]
       }
-    ]
+    ],
+    "@typescript-eslint/explicit-function-return-type": [
+      "error",
+      {
+        "allowExpressions": true
+      }
+]
```

Comment:
This rule means we don't need to add a return type to the custom props in the styled components.
I think it might also be worth adding this to maps, as I think it will remove the need to provide a function return type to our tests which I think is a bit useless anyway 🤷‍♂

Reply:
Agreed, I have also been thinking of talking to you about the return type on react components - probably not needed either - it will always be jsx or falsey.